<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scientistEditDialog</class>
 <widget class="QDialog" name="scientistEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="edit_photo_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_scientist_year_of_death">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_scientist_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_update_scientist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_edit_if_male">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>88</width>
     <height>177</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_scientist_name">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_scientist_year_of_birth">
          <property name="text">
           <string>Year of birth:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_scientist_year_of_death">
          <property name="text">
           <string>Year of death:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_scientist_gender">
          <property name="text">
           <string>Gender:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_add_scientist_photo">
        <property name="text">
         <string>Add photo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_add_scientist_info">
        <property name="text">
         <string>Add info:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="edit_text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_scientist_photo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_edit_if_female">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_edit_window">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_scientist_year_of_birth">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="6">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="label_scientist_name">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_scientist_year_of_birth">
           <property name="text">
            <string>Year of birth:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_scientist_year_of_death">
           <property name="text">
            <string>Year of death:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_scientist_gender">
           <property name="text">
            <string>Gender:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_add_scientist_photo">
         <property name="text">
          <string>Add photo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_add_scientist_info">
         <property name="text">
          <string>Add info:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="edit_scientist_name"/>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="radioButton_edit_if_male">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edit_scientist_year_of_birth"/>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="radioButton_edit_if_female">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="edit_photo_button">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="edit_scientist_year_of_death"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_add_scientist">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Save scientist</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-VLN1-3-Desktop_Qt_5_7_0_MSVC2015_64bit-Debug/icons/save.ico</normaloff>../build-VLN1-3-Desktop_Qt_5_7_0_MSVC2015_64bit-Debug/icons/save.ico</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_scientist_photo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="edit_text">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="toolTip">
      <string>Cancel</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-VLN1-3-Desktop_Qt_5_7_0_MSVC2015_64bit-Debug/icons/cancel.ico.ico</normaloff>../build-VLN1-3-Desktop_Qt_5_7_0_MSVC2015_64bit-Debug/icons/cancel.ico.ico</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+C</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_scientist_name</tabstop>
  <tabstop>edit_scientist_year_of_birth</tabstop>
  <tabstop>edit_scientist_year_of_death</tabstop>
  <tabstop>radioButton_edit_if_female</tabstop>
  <tabstop>radioButton_edit_if_male</tabstop>
  <tabstop>edit_photo_button</tabstop>
  <tabstop>edit_text</tabstop>
  <tabstop>pushButton_add_scientist</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
